### DISCUSSION TITLE
Add a text control for "no items" in the loop grid widget

### DISCUSSION THREAD

**Discussion by vladninja** [elementor/elementor#21268] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, there is no any information about "no items" in the loop grid for end users. In some other widgets, we have a text control which allows using a custom text when there are no results in the loop.

### Describe the solution you'd like

Add a text control for loop grid widget for entering a text message, where there are no items in the loop.

### Describe alternatives you've considered

use some predefined text when no items in the loop grid instead of new control

**Comment by Michaelgimii** [comment]
This option is already available. But it is strange that the developers have hidden it!
This option is only displayed if you enable pagination and select the pagination type to infinite scroll or load on click.

**Reply by areiterer** [reply]
This is the "No more posts" text. If I add "No items were found" there, this will also be shown if I scroll down to the last item.
There should be a different option for when no items were found at all.
